indico/modules/events/reminders/models/reminders.py

```diff
@@ -140,6 +148,59 @@ def all_recipients(self):
         This includes both explicit recipients and, if enabled,
         participants of the event.
         """
+        print("--- EVENT TYPE ------")
+        print(self.event.type)
+        print(self.event.type == "meeting")
+
+        # This should get the persons in the event I presume
+        # But where is the EVENT model located? The event model is in modules/event/models/events.py
+        # An Event Person is associated with an Event using the Event person class. You can loop through it
+        # and get the email from there.
+        # Only Event Person links is enough for the lecture
+        print("---- EVENT PERSON LINKS ------")
+        for pl in self.event.person_links:
+            print(pl.email)
+        
+        # The above thing will not extract the contribution speaker or subcontribution speaker.
+        # We need to loop in and get those
+        contrib_speakers = (
+                ContributionPersonLink.query
+                .filter(
+                    ContributionPersonLink.is_speaker,
+                    ContributionPersonLink.contribution.has(is_deleted=False, event=self.event)
+                )
+                .all()
+            )
+        
+        # TODO (naveen): Learn about SQLAlchemy ORM and how to construct the queries
```

Comment:
These are some notes to myself, will clean it up when I convert the PR from draft :)